plusdeck/cli.py

```diff
@@ -6,14 +6,9 @@
 import logging
 import os
 import sys
-from typing import Any, Callable, Coroutine, List, Literal, Optional
+from typing import Any, Callable, Coroutine, List, Literal, Optional, Self, TypeVar
 import warnings
 
-try:
```

Comment:
I'm concerned that this will break support for older Pythons. Ideas:

1. Fix with typeshed? **DOES NOT WORK**
2. Roll back support for earlier pythons
3. Don't use the Self type

Reply:
It appears Self was introduced in 3.11, and that this pattern is only necessary if I want to support 3.10. 3.10 has other challenges, at least [in crystalfontz](https://github.com/jfhbrook/crystalfontz/pull/47).

I'm deciding right now that 3.10 support is not worth considering at this time, either for plusdeck or crystalfontz. Resolving this as-is, closing the 3.10 issue/PR in crystalfontz, and implementing this same fix there.